.github/workflows/keycloak-scalability-benchmark-on-schedule.yml

```diff
@@ -0,0 +1,27 @@
+name: ROSA Scheduled Create
+
+on:
+  schedule:
+    - cron: '0 7 * * 1-5' # Runs At 07:00 UTC on every day-of-week from Monday through Friday.
+
+jobs:
+
+  checkout:
+    name: ROSA Scheduled Create cluster
+    runs-on: ubuntu-latest
+    steps:
+      - name: Run ClientSecret scalability benchmark
+        run: |
+          gh workflow run -R keycloak/keycloak-benchmark keycloak-scalability-benchmark.yml \
+            -f "scenarioName=authentication.ClientSecret"
```

Comment:
do we want to define this for ClientSecret scenario as well ? `-f "numberOfEntitiesInRealm=100000"`

Reply:
true, but I only see the `numberOfEntitiesInRealm` defined for AuthorizationCode scenario and not for ClientSecret., or am I missing something ? @mhajas

Reply:
`numberOfEntitiesInRealm` defaults to 10000, so it does not need to be added for ClientSecret. Do we want to add it too? For example, if the default changes?